docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -157,6 +160,52 @@ Preparing For a Release
 
    In addition, review the link-check step of the documentation builder to ensure that all the errors (the links with "[broken]") are expected.
 
+#. Update the staging documentation.
+
+   .. TODO: The following steps should be automated
+
+   #. Copy release documentation from ``clusterhq-dev-docs`` to ``clusterhq-staging-docs``.
+
+      .. prompt:: bash $
+
+         gsutil -m rsync -d -r s3://clusterhq-dev-docs/$(python setup.py --version)/ s3://clusterhq-staging-docs/en/${VERSION}/
```

Comment:
For documentation releases (#1092), the `${VERSION}` should in fact be the base release version.